test/new_relic/manticore/instrumentation_test.rb

```diff
@@ -101,16 +101,38 @@ class InstrumentationTest < Minitest::Test
       end
 
       describe "when manticore is used as transport for a database (e.g. in Elasticsearch)" do
-        it "does not create an external request segment" do
-          expect(NewRelic::Agent::External).not_to receive(:start_segment)
+        it "does not deduct manticore time from exclusive database time" do
+          client = ::Manticore::Client.new
+          expect(client.client).to receive(:execute).and_wrap_original do |original, *args|
+            sleep(1)
+            original.call(*args)
+          end
 
           in_transaction do
             NewRelic::Agent::Datastores.wrap("Elasticsearch", "Search", "index_name") do
-              ::Manticore.post(request_uri, body: "data").body
+              client.post(request_uri, body: "data").body
             end
           end
 
+          database_spec = metric_spec_from_specish("Datastore/operation/Elasticsearch/Search")
+          exclusive_time = NewRelic::Agent.instance.stats_engine.to_h[database_spec].total_exclusive_time
+
+          assert_operator(exclusive_time, :>, 1.0)
           assert_metrics_recorded("Datastore/operation/Elasticsearch/Search" => { call_count: 1 })
```

Comment:
As discussed the metric name should be moved to a local var or let.

Reply:
Not sure if that's possible with the minitest integration of rspec-mocks, but we should really disable expecting not implemented methods...